nav2_safety_nodes/README.md

```diff
@@ -0,0 +1,13 @@
+# Navigation Safety Node
+
+The Navigation Safety Node is a safety watchdog node to ensure the robot is acting properly and not about to collide with an obstacle. Typical safety-rated lidars will contain “safety zones” whereas if any sensor points are located in a box around the lidar, then the lidar will send a signal to the robot to stop due to a potential collision.
+
+The node sits below the navigation stack but above the robot controller to do the following:
+
+* Take in the current command velocity from navigation and the most recent laser or RGBD scan.
```

Comment:
This is not what the node does that you have implemented below.

Reply:
Yes, I thought I'm supposed to readme for what overall package does!

Reply:
But it doesn't do that, looking over the PR.

Reply:
Yes, I'll update the readme again